src/components/MessageCardContainerINBOX.js

```diff
@@ -0,0 +1,16 @@
+import { Fragment } from "react";
+import { Link } from "react-router-dom";
+
+import "../../src/styles.css";
+import MessageCard from "./MessageCard";
+
+export default function MessageCardContainerINBOX(){
+	return(
+		<Fragment>
```

Comment:
this is really ugly formatting

Reply:
Thank you so much, we have prettified it.